x-pack/test/reporting/functional/reporting.js

```diff
@@ -204,14 +166,11 @@ export default function ({ getService, getPageObjects }) {
         });
       });
 
-      // TODO Re-enable the tests after removing Phantom:
-      // https://github.com/elastic/kibana/issues/21485
-      describe.skip('Preserve Layout', () => {
+      describe.skip('PNG Preserve Layout', () => {
```

Comment:
There's still `describe.skip` in this branch. Was is supposed to be changed to `describe` ?

Reply:
Yup, likely a bad merge on my part, I'll re-enable.